bash_completion.d/clush

```diff
@@ -1,16 +1,29 @@
 # clush bash completion
 #
 # to install in /usr/share/bash-completion/completions/ or ~/.local/share/bash-completion/completions/
+_clush_command_or_file() {
+	# undo our nospace setting...
+	compopt +o nospace
+
+	# complete either files (copy mode) or commands (if target set)
+	case "$target_set,$mode" in
+	*,copy) _comp_compgen_filedir;;
```

Comment:
Isn't `_comp_compgen_filedir` be a "recent" addition to `bash-completion`? Looks like it's been [introduced](https://github.com/scop/bash-completion/commit/6e8a1546bde205fd7064d9e578f35ec67e2c6840) in 2.12, and I think even el9 only ships `bash-completion` 2.11.
Maybe we need a more backward compatible alternative?

Reply:
oh, good catch!
Looks like that commit just renamed `_filedir` which has been around since forever, but they didn't keep an alias for the old one so I guess we'll need to check..l

It's not too hard to check once at source time and keep the variant to use in a variable, probably better than install time check...

Reply:
Done, the check takes 4~5us on my machine so just checking everytime instead of hogging a variable for this. Tested on debian stable (also bash-completon 2.11) and fedora 41 (bash-completion 2.16)

Reply:
Thanks @martinetd!